flocker/node/agents/test/test_cinder.py

```diff
@@ -61,3 +67,39 @@ def test_verify_false_ca_path(self):
             'verify_ca_path': '/a/path'
         }
         self.assertEqual(_openstack_verify_from_config(**config), False)
+
+
+@implementer(ICinderVolumeManager)
+class FakeCinderClient(object):
+    def delete(self, volume_id):
+        """
+        A no-op delete
+        """
+
+    def get(self, volume_id):
+        """
+        A no-op get.
+        """
+
+
+class CinderDestroyTests(SynchronousTestCase):
+    def test_timeout(self):
+        """
+        ``CinderBlockDeviceAPI.destroy_volume`` raises
+        ``TimeoutException`` if the volume is not removed in 60s.
```

Comment:
This doesn't match the actual timeout hard-coded below (1s) nor the timeout in the implementation.  Maybe "if the volume is not deleted within some time."

(As a side-note, it's also not an deal test insofar as it's somewhat different than real-world usage. On the other hand it's probably the best test one *could* do without massive infrastructure work on OpenStack error injection :grinning: so this is good approach.)